contracts/access/ZNSRoles.sol

```diff
@@ -22,6 +22,11 @@ abstract contract ZNSRoles {
      */
     bytes32 public constant REGISTRAR_ROLE = keccak256("REGISTRAR_ROLE");
 
+    /**
+     * @notice This role is here specifically for the ZNSDomainToken.sol contract
+     */
+    bytes32 public constant DOMAIN_TOKEN_ROLE = keccak256("DOMAIN_TOKEN_ROLE");
```

Comment:
can't we just use Registrar role for this? this way we don't need to upgrade all contracts just for this string...
thoughts?

Reply:
actually... we may have to upgrade all contracts anyway since they all have custom errors now...

Reply:
It's sort of a decision on the "proper" way this should be. It would definitely be the same effect to just assign the `DomainToken` contract the `REGISTRAR_ROLE` but it isn't a registrar, so it felt improper. If we have to upgrade regardless it might make sense to leave it. Let me know what you think